tests/test_definitions/test_expectations.py

```diff
@@ -164,54 +182,65 @@ def pytest_generate_tests(metafunc):
                         if not generate_test:
                             continue
 
-                        if "suppress_test_for" in test and (
-                            (
+                        if "suppress_test_for" in test:
```

Comment:
Since we use `test["suppress_test_for"]` a decent bit below, could we use a `get` statement here?

```python
suppress_test_for = test.get("supprress_test_for")
if suppress_test_for is None or not isinstance(suppress_test_for, list):
    ...

if "sqlalchemy" in suppress_test_for ...
```